odo/backends/sql.py

```diff
@@ -692,6 +692,7 @@ def select_or_selectable_to_frame(el, bind=None, dshape=None, **kwargs):
             parse_dates=datetime_fields,
             dtype=other_dtypes,
             skip_blank_lines=False,
+            escapechar=kwargs.get('escapechar', '\\'),
```

Comment:
This may not be needed anymore. I was playing around with `QUOTE_NONNUMERIC` but that breaks NaT timestamps. I will double check.